internals/cli/env.go

```diff
@@ -25,7 +25,7 @@ var (
 // App represents a command-line application that wraps the
 // kingpin library and adds additional functionality.
 type App struct {
-	Cmd              *cobra.Command
+	Cmd              *CommandClause
```

Comment:
May be a bit weird to suggest, but since it is the root commands, should we rename it to `Root`?
I ask this because I noticed some code with `app.Cmd.Cmd`, which can be confusing. Doing `app.Root.Cmd` may look a bit cleaner. Let me know what you think.

Reply:
Yeah, good point. I think `Root` sounds good.

Reply:
I'll probably simplify this in a refactor PR